CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Changed
+- Switch from Semantic versioning to Cadenza Version based versioning (Starting with Cadenza 10.2)
```

Comment:
Base the version scheme on the Cadenza main version (starting with Cadenza 10.2). New versions have the format <cadenza-version>.<counter>.

Reply:
Done.